docs/workflows/upload-collection.rst

```diff
@@ -1,56 +1,29 @@
 Upload Content
 ==============
 
-Upload an Artifact to Pulp
---------------------------
+Upload a Collection via the REST API
+------------------------------------
 
-This example will upload the ``geerlingguy/ansible-role-postgresql`` role. You can download that
-locally with::
+A collection tarball can be uploaded via the Rest API and the server determines the metadata and
+creates the Collection model.
 
-    curl -L https://github.com/geerlingguy/ansible-role-postgresql/archive/master.tar.gz -o pg.tar.gz
+Suppose you have a file locally named collection.tgz you can upload that to ``pulp_ansible`` with::
 
+    http --form POST :24817/ansible/collections/ file@collection.tgz sha256=abc123
```

Comment:
Maybe drop the sha256 option from this command since you cover it in the next section.

Reply:
Good catch, this was a mistake.